packages/shared/src/storage/GoogleDriveStorage/GoogleDriveStorage.android.ts

```
@@ -0,0 +1,265 @@
+ * - Files stored in hidden scope (application private data)
+ * - Automatic cleanup of temporary files
+ */
+
+const APP_PRIVATE_DATA_SCOPE = 'hidden';
+export class GoogleDriveStorage {
+  /**
+   * Generate temporary file path for Google Drive operations
+   */
+  private buildTempFilePath(fileName: string): string {
+    if (!RNFS) {
+      throw new OneKeyLocalError('File system not available');
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)